wicket-kendo-ui-culture/pom.xml

```diff
@@ -25,6 +25,12 @@
 			<artifactId>wicket-jquery-ui-core</artifactId>
 			<version>6.20.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>com.googlecode.wicket-jquery-ui</groupId>
+			<artifactId>wicket-kendo-ui</artifactId>
+			<version>6.20.1-SNAPSHOT</version>
+			<type>bundle</type>
```

Comment:
If you want to use wicket-kendo-ui at a version that is not yet released (a SNAPSHOT version) you have 3 options:

1) You modify your settings.xml inside your ~/.m2/ folder (hidden .m2 folder in your home folder) to also sync against the  snapshot Nexus repo of wicket-kendo-ui
2) You run mvn install for it to install it into your local repository
3) You modify this project's POM to also look-up inside the snapshot Nexus repo of wicket-kendo-ui

I recommend you go with 1 or 2 (preferably 1)